scripts/python-skywater-pdk/skywater_pdk/netlistsvg-generate.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```

Comment:
Maybe `skywater_pdk/cell/generate/netlist.py`?

Reply:
I moved and renamed the file accordingly.